app.rb

```diff
@@ -0,0 +1,82 @@
+require 'sinatra'
+require_relative 'lib/hangman_rules'
+require_relative 'lib/word_generator'
+require_relative 'lib/words'
+require_relative 'lib/word_formatter'
+
+class App < Sinatra::Base
+
+  enable :sessions
+
+  helpers do
+
+    def rules
+      HangmanRules.new(secret_word, guesses)
+    end
+
+    def letters
+      if session[:letters].nil?
+	session[:letters] = ("a".."z").to_a
+      end
+      session[:letters]
+    end
+
+    def secret_word
+      if session[:secret_word].nil?
+	session[:secret_word] = WordGenerator.new(Words.new).random_word
+      end
+      session[:secret_word]
+    end
+
+    def guesses
+      if session[:guesses].nil?
+	session[:guesses] = {:right_guesses => [], :wrong_guesses => []}
```

Comment:
<JOKE>
Why are the right_guesses on the left?
</JOKE>

Maybe there is a hidden class/object here?

Reply:
🙄